README.md

```diff
@@ -1,55 +1,70 @@
 # cfgov-refresh
 
-This repository contains the redesign-in-progress of consumerfinance.gov. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from Wordpress and Django back-ends to elasticsearch to render the site.
+**Description**: This repository contains the redesign-in-progress of the [consumerfinance.gov](http://consumerfinance.gov) website. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from WordPress and Django back-ends to Elasticsearch to render the site.
 
-### This project is a work in progress
+**Technology stack**:
```

Comment:
I'm late to this party. Was just reviewing the cfgov-refresh README for inspiration in starting up a new project. Shouldn't we genericize this stuff to apply to any Unix-based system?